crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -49,8 +49,20 @@ pub struct Mesh {
 /// # use bevy_render::render_resource::PrimitiveTopology;
 /// fn create_triangle() -> Mesh {
```

Comment:
The name of the function should probably be changed.

Also, I think a parallelogram would make a better example than a rectangle, to show how a texture can stretch, but idk.

Reply:
Agreed - I didn't want the example to be too long so people don't just skip it, so I'll try to add a reference to an example.